### PR TITLE
[Statistics] Verify stat file and insert population and workhours if missing

### PR DESCRIPTION
**Pull request by NicEastvillage**
Depends on #5

In this addition various verification is done on the stats file. Including:
- is it even a city
- does it contain inhabitants and households values (required by ActivityGen)

In case the population and workhour entries are missing, some default values are insert as at least one population bracket is required and at least one opening and closing workhour is required.

In the future we might want to generate the stat file from scratch (#11)

### CODE REVIEW

File: perlin.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def get_perlin_noise(x: float, y: float, base: float, scale: float = 0.005, octa
     :param octaves: the octaves to use when sampling, default is 3
     :return: a normalised float of the sample in noisemap
     """
-    return (noise.pnoise2(x=x * scale, y=y * scale, octaves=octaves, base=base) + 1) / 2
+    return (noise.pnoise2(x=x * scale, y=y * scale, octaves=octaves, base=int(base)) + 1) / 2
```
Comment (falkecarlsen) [inline perlin.py]: Why cast base to int? Signature seems to allow for floats.

**Review by falkecarlsen — APPROVED** [review]
Looks good, except erratic change to get_perlin_noise

File: perlin.py (hunk 1 above)
Comment (NicEastvillage) [reply perlin.py]: Hang on. You told me base should be an int, when I said the noise function didn't work due to an "integer argument expected, got float" error. And it also fixed the problem.

Comment (NicEastvillage) [reply perlin.py]: Just tested it. Can confirm that base has to be an integer.

Comment (falkecarlsen) [reply perlin.py]: Hm, I must've forgotten that. But cool, then this change just also has to be applied to #14.

Comment (NicEastvillage) [reply perlin.py]: I changed the type hints for our functions to be integers as well. See 7c1b745feda92

**Pull request merged**